ci_config.json

```diff
@@ -128,5 +128,13 @@
       "libwayland-bin",
       "libwayland-dev"
     ]
+  },
+  "harfbuzz": {
+    "linux_only": true,
+    "debian_packages": [
+      "ragel",
```

Comment:
I was wondering why we needed this!

Then I realized:

```
warning('You have to install ragel if you are going to develop HarfBuzz itself')
```
Ewwww. That's not what a warning is for.

Reply:
That should be fixed in next harfbuzz release: https://github.com/harfbuzz/harfbuzz/pull/3617.